newsfragments/2718.bugfix

```diff
@@ -0,0 +1,2 @@
+Big (1000x?) performance improvement for selections from reflection tables
```

Comment:
Please qualify the 1000x for very large numbers of serial experiments so as to not mislead more general users

Reply:
i.e. O(100,000) experiments, say

Reply:
Thanks @graeme-winter. Newsfragment updated and button pushed.